docs/json_schemas/kratos_external_idp/v0/provider.json

```diff
@@ -0,0 +1,132 @@
+{
+  "title": "ProviderSchema",
+  "description": "Provider schema for KratosExternalIdp.",
+  "type": "object",
+  "properties": {
+    "unit": {
+      "$ref": "#/definitions/BaseModel"
+    },
+    "app": {
+      "$ref": "#/definitions/KratosExternalIdpProviderData"
+    }
+  },
+  "required": [
+    "app"
+  ],
+  "definitions": {
+    "BaseModel": {
+      "title": "BaseModel",
+      "type": "object",
+      "properties": {}
+    },
+    "SecretBackend": {
+      "title": "SecretBackend",
+      "description": "An enumeration.",
+      "enum": [
+        "relation",
+        "secret",
+        "vault"
+      ]
+    },
+    "Provider": {
+      "title": "Provider",
+      "description": "An enumeration.",
+      "enum": [
+        "microsoft",
+        "apple",
+        "generic",
+        "auth0",
+        "google",
+        "facebook",
+        "github",
+        "gitlab",
+        "slack",
+        "spotify",
+        "discord",
+        "twitch",
+        "netid",
+        "yandex",
+        "vk",
+        "dingtalk"
+      ]
+    },
+    "ExternalIdpProvider": {
+      "title": "ExternalIdpProvider",
+      "type": "object",
+      "properties": {
+        "client_id": {
+          "title": "Client Id",
+          "type": "string"
+        },
+        "client_secret": {
+          "title": "Client Secret",
+          "type": "string"
+        },
+        "secret_backend": {
+          "default": "relation",
+          "allOf": [
+            {
+              "$ref": "#/definitions/SecretBackend"
+            }
+          ]
+        },
+        "provider": {
+          "$ref": "#/definitions/Provider"
+        },
+        "scope": {
+          "title": "Scope",
+          "type": "string"
+        },
+        "provider_id": {
+          "title": "Provider Id",
+          "type": "string"
+        },
+        "jsonnet_mapper": {
+          "title": "Jsonnet Mapper",
+          "type": "string"
+        },
```

Comment:
This could use a description - it's not obvious what this does, nor the format

Reply:
Done. For future reference, since this is autogenerated, I had to add this to the pydantic models.